src/sas/sascalc/data_util/calcthread.py

```diff
@@ -119,30 +111,33 @@ def OnCalcComplete(self,event):
             ...
     """
 
-    def __init__(self, completefn=None, updatefn=None,
-                 yieldtime=0.01, worktime=0.01,
-                 exception_handler=None):
-        """Prepare the calculator"""
-        self.yieldtime     = yieldtime
-        self.worktime      = worktime
-        self.completefn    = completefn
-        self.updatefn      = updatefn
+    def __init__(self,
```

Comment:
This API change inadvertently breaks fitting. `calcthread.py` is also used by `FittingWidget.py` and it passes arguments by keyword on line 2888.

Reply:
Ah, thought I'd caught them all. This is why you don't pass functions around.

Reply:
I've just reverted the changes.